radio/src/pulses/pulses_arm.cpp

```diff
@@ -209,6 +209,8 @@ void setupPulses(uint8_t port)
 #endif
     case PROTO_PPM:
       setupPulsesPPM(port);
+      // 45 ms is the worst case length of the PPM frame length (800 µs between channels and 16 channels
```

Comment:
But the 45 as argument of the scheduleNextMixerCalculation is really in ms. See Crossfire (4) and PXX (9)